### PR TITLE
ENH: added a zenodo json

### PR DESCRIPTION
**Pull request by aburrell**
Added a zenodo json file with creators, access right, and resource type specified.

# Description

Partially addresses #495 

Please include a summary of the change and which issue is fixed. Please also
include relevant motivation and context. List any dependencies that are required
for this change.  Please see ``CONTRIBUTING.md`` for more guidelines.

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

# How Has This Been Tested?

It hasn't not sure how to do so.

**Test Configuration**:
* Operating system
* Version number
* Any details about your local setup that are relevant

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

**Comment by aburrell** [comment]
Could everyone please update their author affiliation and ORCiD in the new .zenodo.json file?  This will ensure citations to the package in the future are properly attributed to you.

@rstoneback @jklenzing @asher-pembroke @jcspence @silviolleite

**Comment by jklenzing** [comment]
@JonathonMSmith

**Comment by aburrell** [comment]
@Gayatri012

**Comment by jcspence** [comment]
@aburrell I've put my updates in #521 - do I need to do anything further?

**Comment by aburrell** [comment]
@jcspence you're set now, thank you!

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      {
+        "affiliation": "University of Texas at Dallas", 
+        "name": "Matthew Depew"
+      }, 
+      {
+        "name": "NHargrave"
+      }, 
+      {
+        "name": "Jonathon Smith"
+      },
```
Comment (jklenzing) [inline .zenodo.json:34]:
```suggestion
      {
        "affiliation": "Catholic University of America, Goddard Space Flight Center",
        "name": "Jonathon Smith"
       "orcid": "0000-0002-8191-4765"
      }, 
```

File: .zenodo.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        "affiliation": "University of Texas at Dallas", 
+        "name": "Matthew Depew"
+      }, 
+      {
+        "name": "NHargrave"
+      }, 
+      {
+        "affiliation": "Catholic University of America, Goddard Space Flight Center",
+        "name": "Jonathon Smith"
+       "orcid": "0000-0002-8191-4765"
```
Comment (jklenzing) [inline .zenodo.json:35]: I got the spacing wrong earlier
```suggestion
        "name": "Jonathon Smith",
        "orcid": "0000-0002-8191-4765"
```

**Pull request merged**